docs/bin/bundle.sh

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
+#
+# Bundle a sample project for download.
+#
+# Usage:
+#   bundle.sh --zip <file> <dir>
+
+set -euo pipefail
+
+function _script_path {
+  local source="${BASH_SOURCE[0]}"
+  while [ -h "$source" ] ; do
+    local linked="$(readlink "$source")"
+    local dir="$(cd -P $(dirname "$source") && cd -P $(dirname "$linked") && pwd)"
+    source="$dir/$(basename "$linked")"
+  done
+  echo ${source}
+}
+
+readonly script_path=$(_script_path)
+readonly script_dir="$(cd -P "$(dirname "$script_path")" && pwd)"
+readonly docs_dir="$(cd "$script_dir/.." && pwd)"
+readonly bundle_dir="$docs_dir/build/bundle"
+
+function _remove_doc_tags {
+  local -r dir="$1"
+  # note: use commands that are compatible with both GNU sed and BSD (macOS) sed
+  find "$dir" -type f -exec sed -i.bak "/tag::[^\[]*\[.*\]/d" {} \; -exec rm -f {}.bak \;
+  find "$dir" -type f -exec sed -i.bak "/end::[^\[]*\[.*\]/d" {} \; -exec rm -f {}.bak \;
+  find "$dir" -type f -exec sed -i.bak "s/\/\/\s<.>//g" {} \; -exec rm -f {}.bak \;
```

Comment:
I don't think this is compatible across both BSD sed and GNU sed.

I don't think `\s` is supported across both (even with extended enabled). Could use `[[:space:]]` but I think we can just expect a literal space and not worry about tab characters. Also, could be useful to match digits specifically, and could allow for more than one (just in case an example gets to 10 callouts). And while we're here, could be useful to strip the leading whitespace as well.

So sticking to basic regex, maybe something like this instead: `"s/ *\/\/ *<[0-9][0-9]*>//g"`

Reply:
Updated to match the new regex you suggested